include/aws/crt/StringView.h

```diff
@@ -7,19 +7,21 @@
 /**
  * string_view is introduced in C++ 17.
  */
-#if __cplusplus >= 201703L
-#    include <string_view>
-#else
+#if __cplusplus < 201703L
```

Comment:
trivial/debatable: simpler to just do 1 ifdef instead of 3?
```
#ifdef __cplusplus >= 201703L
#include <string_view>
namespace Aws
{
    namespace Crt
    {
        using StringView = std::string_view;
    }
}
#else
...rest of file...
#endif
```

Reply:
I'd made this comment on the original PR.
I went back to find a link to it, so I could say "I told you so".
I discovered that I'd never actually submitted the comment.
🤦‍♂️🤦‍♂️🤦‍♂️